src/pages/artist/components/Related.js

```diff
@@ -1,10 +1,49 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import Artist from '../../home/atoms/Artist'
```

Comment:
If you are using the same `Artist` component from `Home`, then it doesn't really go under `/home/atoms`, because it's no longer a component specific only for Home page.

I think you should place it under shared "atoms" or "components", so move it from the `/home/atoms`.